### ISSUE TITLE
Competition Proposal: [1610.07882] Maxmin convolutional neural networks for image classification

### ISSUE THREAD

**Issue by karandesai-96** [samim23/GitXiv#57]
The paper specifies that they would be releasing the code on Github (CUDA Implementation). But it is not found on Github yet.

This paper was accepted in ICIP 2016 and introduces a very simple modification to the traditional convolutional layer for boosting accuracy. I have implemented the paper using Keras, and even got a PR with Caffe implementation. The results are reproduced as they are mentioned.

Paper: http://ieeexplore.ieee.org/document/7533046/
Github Repo: https://github.com/karandesai-96/maxmin-cnn
